src/blockchain/log-processors/legacy-rep-contract/transfer.ts

```diff
@@ -3,15 +3,17 @@ import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../../types";
 
 export function processTransferLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
-  const dataToInsert: {} = {
-    transactionHash:  log.transactionHash,
-    logIndex:         log.logIndex,
-    sender:            log.from,
-    recipient:         log.to,
-    token:             log.address,
-    value:             log.value,
-    blockNumber:      log.blockNumber,
-  };
+  db.transacting(trx).insert({
+    transactionHash: log.transactionHash,
+    logIndex:        log.logIndex,
+    sender:          log.from,
+    recipient:       log.to,
+    token:           log.address,
+    value:           log.value,
+    blockNumber:     log.blockNumber,
+  }).into("transfers").asCallback(callback);
+}
 
-  db.transacting(trx).insert(dataToInsert).into("transfers").asCallback(callback);
+export function processTransferLogRemoval(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
+  db.transacting(trx).from("transfers").where({ transactionHash: log.transactionHash, logIndex: log.logIndex, blockNumber: log.blockNumber, token: log.address }).del().asCallback(callback);
```

Comment:
Could just use transactionHash/logIndex ?

Reply:
That seems reasonable.  Is that guaranteed to be unique?

Reply:
```CREATE TABLE transfers (
      "transactionHash" varchar(66) NOT NULL,
      "logIndex" integer NOT NULL CONSTRAINT "nonnegativeLogIndex" CHECK ("logIndex" >= 0),
      sender varchar(66) NOT NULL,
      recipient varchar(66) NOT NULL,
      token varchar(66) NOT NULL,
      value numeric NOT NULL CONSTRAINT "positiveValue" CHECK (value > 0),
      "blockNumber" integer NOT NULL CONSTRAINT "positiveBlockNumber" CHECK ("blockNumber" > 0),

      UNIQUE("transactionHash", "logIndex")

    );
```